projekt_do_pracy_na_labach/fiszki_backend/db_driver.py

```diff
@@ -4,7 +4,7 @@
 import os 
 
 
-db_url = os.environ.get("DATABASE_URL", "mongodb://localhost:27017")
+db_url = os.environ.get("DATABASE_URL", "mongodb://siec1:27017")#localhost
```

Comment:
Hmm, celem labów było to by nie musieć zmieniać zależności w kodzie, także prosiłbym o jakiś komentarz czemu to tu jest XD